search-services/latest/install/options.md

```diff
@@ -201,9 +201,9 @@ This task assumes you have:
 
     ```text
     index.subsystem.name=solr6
-    solr.secureComms=secret
-    solr.port=8983
-    solr.sharedSecret=password
+    mutual-TLS-solr.secureComms=https
```

Comment:
I think this is still wrong.

Reply:
Right Ive removed the mutual-TLS-solr.secureComms=https

Reply:
@tpage-alfresco can you let me know the correct parameters here please?
im not sure what they are.
cheers.